backend/static/js/keywordSearch.js

```diff
@@ -0,0 +1,76 @@
+// this function removes the highlight
```

Comment:
A minor comment: Our comment style is normal sentences (capitalized and with a full stop). I know that a lot of the qlever-ui does not follow this rule yet. That's becase it's old. But nor reason not to stick to this for all new code.

Reply:
Ah, good to know!
I generated some docstrings with chatGPT.
Do the new comments satisfy the comment style?

```
/**
 * Removes highlighting from the input string by replacing all occurrences of
 * <span> elements with the class "keyword-search-highlight" with their inner content.
 *
 * @param {string} input_str - The input string possibly containing highlighted spans.
 * @returns {string} - The input string with highlighting removed.
 */
```

Reply:
We use ordinary `// ...` comments with full sentences. When referring to variable or class names in a comment, we put it in `...`. See the code on https://github.com/ad-freiburg/qlever for many examples of this (it's also not 100% consistent there, but 90% of the comments follow this convention).